src/components/data-management/ProjectDetails.jsx

```diff
@@ -575,25 +582,50 @@ const ProjectDetails = ({ width, height }) => {
     router.push(analysisPath.replace('[experimentId]', experimentId));
   };
 
+  const pipelineHasRun = (experimentId) => (
+    experiments[experimentId]?.meta?.pipeline?.status === pipelineStatus.SUCCEEDED
+  );
+
   const DownloadDataMenu = (
     <Menu>
-      <Menu.Item disabled>
-        <Tooltip title='Coming soon!' placement='left'>
+      <Menu.Item disabled key='download-raw-seurat'>
+        <Tooltip title='Feature coming soon!' placement='left'>
           {/* <Tooltip title='Samples have been merged' placement='left'> */}
           Raw Seurat object (.rds)
         </Tooltip>
       </Menu.Item>
-      <Menu.Item disabled>
-        <Tooltip title='Coming soon!' placement='left'>
-          {/* <Tooltip title='Data Processing filters have been applied' placement='left'> */}
+      <Menu.Item
+        key='download-processed-seurat'
+        disabled={
+          activeProject?.experiments?.length > 0
+          && !pipelineHasRun(activeProject.experiments[0])
+        }
+      >
+        <Tooltip
+          title={
+            activeProject?.experiments?.length > 0
+            && pipelineHasRun(activeProject.experiments[0])
+              ? 'With Data Processing filters and settings applied'
+              : 'Launch analysis to process data'
+          }
+          placement='left'
+          onClick={() => {
+            // Change if we have more than one experiment per project
+            const experimentId = activeProject.experiments[0];
```

Comment:
shouldn't this be `activeProject?.experiments[0]`

Reply:
Changed

Reply:
The menu item is now disabled if there is no experimentId, so the click does not work if there is no experimentId.

Reply:
change accordingly